src/main/riddl/Members/members.riddl

```diff
@@ -127,6 +127,60 @@ context MembersContext  is {
             }
         }
 
+        state Draft is {
+          fields {
+            info: MembersContext.Info,
+            meta: MembersContext.MetaInfo
+          }
+
+          handler DraftMemberHandler {
+            on command RegisterMember {then error "Member has already been registered."}
+            on command UpdateMemberStatus {
+              example DraftMemberToActiveMember is {
```

Comment:
Alex and I have had several conversations about Members registering themselves. I think it is a common use case. I don't think the riddl needs to change here, but we should be explicit that the RegisteringMember may be "self" or the entity's own ID.

Reply:
@miguelTaningcoYoppworks - probably all that is needed to address James' comment is some documentation on how the RegisterMember message definition

Reply:
Will add documentation! Out of curiosity, does that mean that the RegisterMember command's registeringMember field must now be optional? I'm not too sure how it works if the member is registering themselves and couldn't fill in the registeringMember field.

Reply:
No, not optional. It means that the value for regtieringMember is the same Id(Member)  as the member being registered. And, that's okay.  The documentation should say as much.

Reply:
Added explicitly in RegisterMember command ^

Reply:
Yes, that's correct.